app/views/admin/calendars/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>New Calendar</h1>
```

Comment:
This replaces the new calendar area functionality - at the moment, the link from the calendars index page specifies **regional** - which is what we are really supporting for new calendars at the moment, but logic is there if required - we could disable completely if we wanted, i.e. removing the input radio buttons